src/__experimental__/components/input-icon-toggle/__snapshots__/input-icon-toggle.spec.js.snap

```diff
@@ -16,14 +55,21 @@ exports[`InputIconToggle when initiated with children renders as expected 1`] =
   -ms-flex-pack: center;
   justify-content: center;
   margin-right: -11px;
+  margin-left: -11px;
   width: 40px;
 }
 
 <span
   className="c0"
   size="medium"
-  type="settings"
 >
-  mock content
+  <span
+    className="c1"
```

Comment:
This has a few invalid props.

- `className`
- `disabled`
- `fontSize`
- `type`

Please can you investigate?

Reply:
This snapshot is actually no longer required as InputIconToggle is no longer meant to be rendering passed children.
`span` with invalid props you've mentioned is actually an `Icon` and that's due to the fact, that `Icon` has these defined as default props (and it was like that before my PR)